smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/providers/connectors/WorkerPoolRegistry.java

```diff
@@ -36,7 +38,10 @@
 @ApplicationScoped
 public class WorkerPoolRegistry {
     private static final String WORKER_CONFIG_PREFIX = "smallrye.messaging.worker";
+    @Deprecated(since = "4.8.0")
     private static final String WORKER_CONCURRENCY = "max-concurrency";
+    private static final String WORKER_POOL_SIZE = "pool-size";
```

Comment:
actually, max-concurrency was better.
virtual threads are not pooled, so pool size does not make sense.

Reply:
I wasn't clear enough. With this change, there wouldn't be a need for a `pool-size` or `max-concurrency` config to limit the concurrency with virtual threads. 

For example:
```java
@RunOnVirtualThread
@Blocking(maxConcurrency=1000)
@Incoming("channel")
void process(String payload) {
    //
}
```

For Vert.x workers:
```java
@Blocking(value="my-custom-pool", maxConcurrency=10, ordered=false)
@Incoming("channel")
void process(String payload) {
    //
}
```
with
```properties
smallrye.messaging.worker.my-custom-pool.pool-size=10
smallrye.messaging.worker.my-custom-pool.max-execute-time=30s
```